### PR TITLE
osbuild-image-tests: ignore LVM UUID

### PR DESCRIPTION
**Pull request by achilleas-k**
Unconditionally ignore the UUID of the LVM container.  We don't control this value so it isn't stable and makes our image tests fail.

### CODE REVIEW

File: cmd/osbuild-image-tests/main_test.go [hunk 1]
```diff
@@ -130,6 +130,22 @@ func deleteTmpfilesdPathFromImageInfoReport(imageInfoReport interface{}, path st
...
 
+// Replace the UUID of the root LVM container with a static one.  We do not
+// control this value and so it is not stable across image builds.
+func replaceLVMUUID(imageInfoReport interface{}, path string) {
+	imageInfoMap := imageInfoReport.(map[string]interface{})
+	partitions, exists := imageInfoMap["partitions"]
+	if exists {
+		for _, partition := partitions.([]interface{}) {
+			partitionInfoMap := partition.(map[string]interface{})
+			if islvm, exists := partitionInfoMap["lvm"]; exists && islvm.(bool) == true {
```
Comment (lavocatt) [inline cmd/osbuild-image-tests/main_test.go:141]: I think the correct name for the attribute is `LVM2_member`.

**Review by lavocatt — CHANGES_REQUESTED** [review]
Thanks for the quick fix!

File: cmd/osbuild-image-tests/main_test.go (hunk 1 above)
Comment (lavocatt) [reply cmd/osbuild-image-tests/main_test.go:141]: (at least it's what I've used in my python version of this)

Comment (achilleas-k) [reply cmd/osbuild-image-tests/main_test.go:142]: I'm checking the `"lvm"` boolean of the partition
```json
      {
        "bootable": false,
        "fstype": "LVM2_member",
        "label": null,
        "lvm": true,
        "lvm.vg": "rootvg",
        "lvm.volumes": {
          "optlv": {
            "fstype": "xfs",
            "label": null,
            "uuid": "6e4ff95f-f662-45ee-a82a-bdf44a2d0b75"
          },
          "rootlv": {
            "fstype": "xfs",
            "label": "root",
            "uuid": "0194fdc2-fa2f-4cc0-81d3-ff12045b73c8"
          }
        },
        "partuuid": "6264D520-3FB9-423F-8AB8-7A0A8E3D3562",
        "size": 10094624256,
        "start": 642777088,
        "type": "E6D6D379-F507-44C2-A23C-238F2A3DF928",
        "uuid": "9UzqmJ-LgKG-Qlid-YeYg-vVnd-Z3dc-V20Ozc"
      },
```

Comment (lavocatt) [reply cmd/osbuild-image-tests/main_test.go:142]: Cool that it works the same! It's even neater with this boolean.

**Comment by jrusz** [comment]
Thanks for this fix! I'm running the image-tests now, will report back with results soon.

**Review by thozza — APPROVED** [review]
LGTM, thanks!

**Review by jrusz — APPROVED** [review]
qcow2_customize tests passed, thanks!

**Review by lavocatt — APPROVED** [review]

**Pull request merged**